pkg/webservice/handlers.go

```diff
@@ -60,6 +60,12 @@ const (
 	NodeDoesNotExists        = "Node not found"
 )
 
+var allowedAppStates = map[string]bool{
```

Comment:
Pardon me, why only partial states are allowed?

Reply:
@laysfire could you share the discussion to me? Also, adding the comment here can helper readers in the future :)